frontend/src/components/dasDataTable.js

```diff
@@ -112,7 +112,8 @@ export default class RoomDataTable extends React.Component {
       .catch((error) => {
         this.setState({errorMessage: error.message});
         if (error.response) {
-          if (error.response.status === '401') {
+	  console.log("AAAAA " + error.response.status);
```

Comment:
Um, wat?
I wouldnt know what this means in the log.

Reply:
Ahahaha this is the gadget equivalent of a `printf`. I'll remove it.